Jenkinsfile

```diff
@@ -1,12 +1,7 @@
-properties([buildDiscarder(logRotator(numToKeepStr: '20'))])
-node('maven-11') {
-    checkout scm
-    timeout(time: 1, unit: 'HOURS') {
-        // TODO Azure mirror
-        ansiColor('xterm') {
-            withEnv(['MAVEN_OPTS=-Djansi.force=true']) {
-                sh 'mvn -B -Dstyle.color=always -ntp clean verify'
-            }
-        }
-    }
-}
+// Builds a module using https://github.com/jenkins-infra/pipeline-library
+buildPlugin(useAci: true, configurations: [
+        [ platform: "linux", jdk: "8" ],
+        [ platform: "windows", jdk: "8" ],
```

Comment:
Could we drop this? General recommendation is to not bother running every combination as windows issues are generally windows specific and not jdk specific.

Reply:
doesn't matter really if you prefer that's fine.
I only wanted to test on linux AND windows to avoid such situation https://github.com/jenkinsci/plugin-pom/pull/506 But even this wouldn't have detected it, at least it may detect some issues on windows.